shared/common-adapters/tab-bar.js.flow

```diff
@@ -4,41 +4,40 @@ import React, {Component} from 'react'
 import Avatar from './avatar'
 import type {Props as IconProps} from './icon'
 
-export type ItemProps = {
-  tabBarButton?: React$Element,
-  label?: string,
+export type Props = $Shape<{
```

Comment:
W/o the $Shape<> modifier, flow wasn't performing much type checking, which allowed incorrect props to slip through.